test/jdk/com/sun/crypto/provider/Cipher/AES/TestAESCipher.java

```diff
@@ -73,33 +80,41 @@ public static void main(String argv[]) throws Exception {
         }
     }
 
+    static byte[] makeData(int len) {
+        int i = 0;
+        byte[] b = new byte[len];
+        while (i < len) {
+            b[i] = (byte)(i % 255);
+            i++;
+        }
+        return b;
```

Comment:
I wanted consistent data because it's harder to solve crypto failures when you cannot see consistent pattern in the data.  For example, the GCM case where the ciphertext is correct, but the tag is wrong.  It is easier to know that the problem is in GHASH and not GCTR.
I don't see any negatives to having similar data.

Reply:
Might as well just hardcode the data then. Saves time to re-generate the similar bytes over and over. Not seeing much benefit of this method. Not a big deal if you intentionally use similar data.